### ISSUE TITLE
Auto-backup ate all disk space on NFS mount 

### ISSUE THREAD

**Issue by jvstein** [mealie-recipes/mealie#796]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.

### What is the issue you are experiencing?

I'm running mealie with portions of the `/data` folder mounted over NFS. During a recent auto-backup, Mealie created a 15GB  backup file, chewing through the available disk space I had allocated to it.

```
$ ls -l /app/data/backups/Auto_*.zip
-rw-r--r-- 1 user group 15413583872 Nov 12 00:14 /app/data/backups/Auto_2021-Nov-12.zip
```

When I list the contents of the backup file with `7z l`, I see an [un-closed `.nfs000000000110b7ca0030488c` file](https://serverfault.com/a/234727/47594) in the backup. This makes me suspect Mealie isn't closing file handles properly after an image is uploaded.

The relevant `double-chocolate-chip-cookies` recipe was created by exporting & importing another recipe in the system and then adding an image via the mobile web interface.

```
$ 7z l Auto_2021-Nov-12.zip

7-Zip [64] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21
p7zip Version 16.02 (locale=en_US.UTF-8,Utf16=on,HugeFiles=on,64 bits,8 CPUs Intel(R) Core(TM) i7-10610U CPU @ 1.80GHz (806EC),ASM,AES-NI)

Scanning the drive for archives:
1 file, 15413583872 bytes (15 GiB)

Listing archive: Auto_2021-Nov-12.zip

--
Path = Auto_2021-Nov-12.zip
Type = zip
ERRORS:
Headers Error
Unconfirmed start of archive
WARNINGS:
There are data after the end of archive
Physical Size = 2509815
Tail Size = 15411074057

   Date      Time    Attr         Size   Compressed  Name
------------------- ----- ------------ ------------  ------------------------
2021-11-12 00:00:00 D....            0            0  comments
2021-11-12 00:00:00 D....            0            0  notifications
2021-11-12 00:00:02 D....            0            0  recipes
2021-11-11 23:59:58 D....            0            0  templates
2021-11-12 00:00:02 D....            0            0  themes
2021-11-11 23:59:58 D....            0            0  users
2021-11-12 00:00:02 .....          639          198  themes/themes.json
2021-11-12 00:00:02 D....            0            0  recipes/double-chocolate-chip-cookies
2021-10-30 08:59:10 D....            0            0  recipes/double-chocolate-chip-cookies/assets
2021-11-12 00:00:02 D....            0            0  recipes/double-chocolate-chip-cookies/images
2021-10-30 10:33:32 .....      2508196      2508966  recipes/double-chocolate-chip-cookies/images/.nfs000000000110b7ca0030488c
------------------- ----- ------------ ------------  ------------------------
2021-11-12 00:00:02            2508835      2509164  2 files, 10 folders

Warnings: 1

Errors: 1
```

### Deployment

Docker (Linux)

### Deployment Details

Kubernetes, with NFS volume mounted into the container.

### Mealie Version

0.5.2

**Comment by hay-kot** [comment]
Thank you so much for a detailed report. This has been a problem for a few other users as well. I've been scratching my head, but I think this gives me enough to go on. Will be digging into it this weekend!

**Comment by hay-kot** [comment]
Once the PR below is merged auto-backups will be disabled by default. A workaround until this is tracked down and fixed. 

- #853

**Comment by kushfest** [comment]
I just noticed I'm having a similar issue. I'm on the current beta and the my backup file is 3GB, which is weird, because my recipe data totals to less than 100MB. Also strange...when I extract the 3GB backup zip, the folder it generates is only 100MB. Is something broken in the process to create the zip file?

**Comment by hay-kot** [comment]
@kushfest

If you're running the v1 beta, can you submit a new issue specifying you problem?

The code that caused this issue no longer exists and has been completely rewritten.

If you're still experiencing a problem we'll want to get it fixed. Thanks!

**Comment by kushfest** [comment]
Thanks for the quick response! I'll do that now.
